controllers/comments.controllers.js

```diff
@@ -33,3 +34,18 @@ exports.postComments = (req, res, next) => {
       next(err);
     });
 };
+
+exports.removeComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  
+  checkCommentExists(comment_id)
+    .then(() => {
+      return deleteComment(comment_id);
+    })
+    .then((deletedComment) => {
+      res.status(204).send();
```

Comment:
If not sending back a body, we could opt to use the sendStatus() method instead (e.g. res.sendStatus(204)). This way is totally fine too though, I'm more saying this just for interests' sake 😄